packages/manager/apps/web/client/app/domain/webhosting/order/steps/duration/domain-webhosting-order-steps-duration.controller.js

```diff
@@ -20,11 +20,5 @@ export default class {
 
   updateDuration() {
     this.stepper.cartOption.offer.pricing.duration = this.duration;
-
-    // Necessary as the update of stepper.currentIndex
-    // has to be made after $digest cycle is finished
-    this.$timeout(() => {
```

Comment:
As you removed this, you can remove the constructor (no need since it would be empty)

Reply:
Good catch, thks!